src/components/atoms/Avatar/Avatar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Avatar.scss'
+
+const Avatar = ({ image, alt, size, className, style, ...props }) => (
+  <img
+    src={image}
+    alt={alt}
+    className={`cg-avatar ${className}`}
+    style={{ width: size, height: size, ...style }}
+    {...props}
+  ></img>
```

Comment:
Here we can remove the </img> and make it just <img /> like Self-Closing Tag, since there is no children in this.